### ISSUE TITLE
Mylar having issues with Elephantmen 2261 series on 32p

### ISSUE THREAD

**Issue by Lusiphur** [evilhero/mylar#2298]
Mylar can't find these, even if I put the series id in.

For Elephantmen 2261 : The Death of Shorty 

21-Jun-2019 21:03:12 - DEBUG   :: mylar.NZB_SEARCH.1074 : SEARCH-QUEUE : [CONV] pubdate: 2018-09-20  < storedate: 2018-10-09
21-Jun-2019 21:03:12 - DEBUG   :: mylar.NZB_SEARCH.1075 : SEARCH-QUEUE : Thu, 20 Sep 2018 00:38:28 +0000 is before store date of 2018-10-10. Ignoring search result as this is not the right issue.

For Elephantmen 2261 : The Pentalion Job (file is in my local lookup path but it doesn't pick it up)
21-Jun-2019 21:05:07 - DEBUG   :: mylar.listFiles.146 : Thread-18 : [FILENAME]: Elephantmen - The Pentalion Job 002 (2019) (digital) (Son of Ultron-Empire).cbr
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.226 : Thread-18 : [CORRECTION] Sub-directory found. Altering path configuration.
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.228 : Thread-18 : [SUB-PATH] Checking Folder Name for more information.
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.230 : Thread-18 : [SUB-PATH] Original Path : /hyena_pool/Incoming/Comics
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.231 : Thread-18 : [SUB-PATH] Sub-directory : /hyena_pool/Incoming/Comics/0-Day Week of 2019.06.19
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.244 : Thread-18 : [SUB-PATH] subpath set to : 0-Day Week of 2019.06.19
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.287 : Thread-18 : Scanner group tag discovered: Son of Ultron-Empire
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.394 : Thread-18 : NEWLY SPLIT REORGD: ['Elephantmen', '-', 'The', 'Pentalion', 'Job', '002', '(2019)', '(digital)']
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.429 : Thread-18 : checking date : 2019
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.432 : Thread-18 : date: 2019
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.759 : Thread-18 : datecheck: [{'date': '2019', 'position': 6, 'mod_position': 36}]
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.765 : Thread-18 : sctd: 2019
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.774 : Thread-18 : year verified as : 2019
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.780 : Thread-18 : year verified as: 2019
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.811 : Thread-18 : highest_series_position: 6
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.816 : Thread-18 : possible_issuenumbers: [{'position': 5, 'validcountchk': False, 'number': '002', 'mod_position': 32}]
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.822 : Thread-18 : hyphen located at position: 12
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.824 : Thread-18 : yearposition: 6
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.859 : Thread-18 : issue number :002
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.888 : Thread-18 : issue_position: 5
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.905 : Thread-18 : issue verified as : 002
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.1016 : Thread-18 : ALT-SERIES NAME [ISSUE TITLE]: Elephantmen [The Pentalion Job]
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.1032 : Thread-18 : sf_highest_series_pos: ['Elephantmen', '-', 'The', 'Pentalion', 'Job']
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.1077 : Thread-18 : series title possibly: Elephantmen - The Pentalion Job
21-Jun-2019 21:05:07 - DEBUG   :: mylar.parseit.1090 : Thread-18 : Alternate series / issue title: Elephantmen [The Pentalion Job]

Mylar Version: development
-- git build 0af13913eaf120ced01bddeafb23ceedbcdfc0e8.
Python Version : 2.7.12
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04.6 LTS"

**Comment by barbequesauce** [comment]
Death of Shorty appears to be a CV data issue - not much we can do there. If you put it in your library directory, does file checker recognize it?

I can’t find Pentalion on 32p - what id number are you using?

**Comment by Lusiphur** [comment]
The file checker does not recognise the Death of Shorty.

As for The Pentalion Job, I have that on my local disk from a 0-day pack. The file checker is not seeing it either.

**Comment by evilhero** [comment]
Your files aren't named as per the CV series title. Either add in the name as an Alternate Search Name or change the name of the file to be identical to what the CV title is. 

tldr; you can't find apples if you're searching for oranges.
